srcpkgs/cutter/template

```diff
@@ -1,16 +1,30 @@
 # Template file for 'cutter'
 pkgname=cutter
-version=1.12.0
-revision=2
-build_wrksrc=src
-build_style=qmake
-configure_args="CUTTER_EXTRA_PLUGIN_DIRS=/usr/lib/RadareOrg/Cutter/plugins"
-hostmakedepends="pkg-config qt5-qmake qt5-host-tools radare2"
-makedepends="capstone-devel python3-devel qt5-declarative-devel
- qt5-location-devel qt5-svg-devel radare2"
+version=2.0.5
+revision=1
+archs="i686 x86_64"
+build_style=cmake
+configure_args="-DCUTTER_EXTRA_PLUGIN_DIRS=/usr/lib/rizin/Cutter/plugins
+ -DCUTTER_ENABLE_PYTHON=ON -DCUTTER_ENABLE_PYTHON_BINDINGS=OFF
+ -DCUTTER_ENABLE_GRAPHVIZ=ON -DCUTTER_ENABLE_KSYNTAXHIGHLIGHTING=ON
+ -DCUTTER_USE_BUNDLED_RIZIN=OFF -DCUTTER_USE_ADDITIONAL_RIZIN_PATHS=OFF
+ -DCUTTER_ENABLE_CRASH_REPORTS=OFF"
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools meson git"
+makedepends="capstone-devel python3-devel qt5-declarative-devel qt5-location-devel
+ qt5-svg-devel graphviz-devel libshiboken2-devel libpyside2-python3-devel
+ syntax-highlighting-devel rizin-devel"
 short_desc="GUI for radare2 written in C++ and QT"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
 license="GPL-3.0-only"
-homepage="https://github.com/radareorg/cutter"
-distfiles="https://github.com/radareorg/${pkgname}/archive/v${version}.tar.gz"
-checksum=868213d2ea0b4a29b9b03c9b605c2b2155c4c03b62735a9bd376a0dadcb4fe1b
+homepage="https://cutter.re"
+distfiles="https://github.com/rizinorg/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=c206cc72443b989b7306cbc1ec5c85206a5feb5e7dc8621fe4f03876b9e447ab
+
+do_fetch() {
+	if [ ! -e "$wrksrc" ]; then
+		git clone --recurse-submodules "https://github.com/rizinorg/${pkgname}" $wrksrc
+	fi
+	cd $wrksrc
+	git checkout "v${version}"
+	git submodule update --init --recursive
+}
```

Comment:
They don't offer release source tarballs. I've several other packages in my pipeline with the same issue. Looks like package maintainers don't know (or don't care) that github does not include submodules into automatically generated tarballs. Probably because of things like flatpack. But other distributions don't seem to care as well: They just fetch the source from github directly, like I did.

Reply:
There is an issue for this which is unanswered since ~2 months: https://github.com/rizinorg/cutter/issues/2878

I guess they have more important stuff to do and we have to go with the "ugly" solution.

Reply:
That's unfortunate, but could you do something similar to #32199?